### PR TITLE
refactored for multiple output types; also did single word/phrase-emo…

### PR DESCRIPTION
**Pull request by TennisGazelle**
…ji replacement

### CODE REVIEW

File: espell.py [hunk 1]
```diff
@@ -1,20 +1,34 @@
 #!/usr/bin/env python3
-
+import sys
 import json
 import random
 
+def getEmojifiedVersion(message):
+    with open('/Users/dlopez7/dev/emoji-slack/translator.json') as t_file:
```
Comment (asnewman) [inline espell.py]: is this going to work on everyone's machine?

File: espell.py [hunk 2]
```diff
@@ -1,20 +1,34 @@
...
+            if letter.lower() in translator:
+                options = translator[letter.lower()]
+                newMessage += random.choice(options)
+            else:
+                newMessage += letter
+        
+        return newMessage
+
+def pepperWithEmojis(message):
+    with open('/Users/dlopez7/dev/emoji-slack/emoji-list.json') as t_file:
```
Comment (asnewman) [inline espell.py]: Here too

File: espell.py (hunk 1 above)
Comment (TennisGazelle) [reply espell.py]: good catch, thanks

File: espell.py (hunk 2 above)
Comment (TennisGazelle) [reply espell.py]: got it

**Pull request merged**